readme.md

```diff
@@ -359,6 +359,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [pretty-error](https://github.com/AriaMinaei/pretty-error) - Errors with less clutter.
 - [humanize](https://github.com/taijinlee/humanize) - Data formatter for human readability.
 - [node-read](https://github.com/bndr/node-read) - Extract readable content from any page.
+- [read-art](https://github.com/Tjatse/node-readability) - Scrape/Crawl article from any site automatically. Make any web page readable, no matter Chinese or English.
```

Comment:
How is this different/better than `node-read`? (this should really be written in the `read-art` readme).